server/src/main/resources/db/common/V86__alter_feedback_answer_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE feedback_answers
+ADD CONSTRAINT feedback_answers_uniqueconstraint UNIQUE (question_id, request_id);
```

Comment:
This migration needs to be renumbered to V91

Reply:
fixed